patch_subset/sparse_bit_set.cc

```diff
@@ -76,6 +76,7 @@ StatusCode SparseBitSet::Decode(string_view sparse_bit_set, hb_set_t* out) {
   uint64_t node_base_factor = leaf_node_size >> kBFNodeSizeLog2[branch_factor];
   vector<uint32_t> node_bases{0u};  // Root node.
   vector<uint32_t> next_level_node_bases;
+  vector<uint32_t> pending_codepoints;
```

Comment:
prefer to have this be vector<hb_codepoint_t> which matches what  the set api takes.

Reply:
Done.